web/src/unread_ops.js

```diff
@@ -56,7 +56,8 @@ export function confirm_mark_all_as_read() {
     });
 }
 
-function bulk_mark_messages_as_read(narrow, args = {}) {
+function bulk_mark_messages_as_read_or_unread(narrow, args = {}, operation = "read") {
+    const op = operation === "read" ? "add" : "remove";
```

Comment:
It might be cleaner to rename this to `bulk_update_read_flags_for_narrow(narrow, args, "add|remove") rather than having this conversion from "read"/"unread" to "add"/"remove".

Reply:
I see you want to use this as a variable below, but that doesn't workwith i18n; the string "read" will always be in English because that's the language the code is written in.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html to educate yourself a bit more about this consideration, but the correct answer likely involves two copies of those strings.

Reply:
Updated it!